### ISSUE TITLE
No amplifyconfiguration file

### ISSUE THREAD

**Issue by nkb1337** [aws-amplify/amplify-cli#4747]
Hello,

I created a new iOS project following the official guide. I successfully added Auth and pushed everything to the cloud.

However, the CLI doesn't generate any amplifyconfiguration for me, however the awsconfiguration gets created, any suggestions?

Also another thing, maybe I am just too stupid / or confused, I uninstalled and then freshly installed the amplify / cli package.

amplify -v
1.1.7

However npm list -g --depth=0 shows me
@aws-amplify/cli@4.22.0

Am I  on the correct version or is this the issue why my config file is not generated?

Best Regards

**Comment by jhockett** [comment]
Hi @nkb1337, `amplify -v` should show you 4.22.0 if you just installed the latest version.

Was version 1 of the CLI previously installed on your machine? If you uninstall the CLI and type `amplify -v`, what happens?

**Comment by nkb1337** [comment]
> Hi @nkb1337, `amplify -v` should show you 4.22.0 if you just installed the latest version.
> 
> Was version 1 of the CLI previously installed on your machine? If you uninstall the CLI and type `amplify -v`, what happens?

Hi thnaks for the reply and sorry for my late response.

After I uninstalled it via npm it showed nothing. I even checked for any leftovers but it seems that iwas correctly uninstalled.

I found the amplify configuration things I needed on github in some other questions.

However, since this is not related to the amplify cli and an issue on my side I'll close this and fix this npm issue.

Best Regards and thanks for the help
Niko
